frontend/src/components/summary/group/ECEWESummary.vue

```diff
@@ -49,145 +49,125 @@
-                    <v-textarea
-                      placeholder="Required"
-                      :model-value="getFundingModel(ecewe?.fundingModel)"
-                      class="summary-value"
-                      density="compact"
-                      flat
-                      variant="solo"
-                      hide-details
-                      readonly
-                      no-resize
-                      rows="3"
-                      :rules="rules.required"
-                    />
-                  </v-col>
-                </v-row>
-                <v-row
-                  v-if="
-                    ecewe?.fundingModel === fundingModelTypeList[1].id ||
-                    ecewe?.fundingModel === fundingModelTypeList[2].id
-                  "
-                  no-gutters
-                >
-                  <v-col cols="12">
-                    <span class="summary-label pt-3">
-                      I confirm that my organization/facilities pay the Joint Job Evaluation Plan (JJEP) wage rates or,
-                      if a lesser amount, a side agreement is being concluded to implement the ECE Wage Enhancement.
-                    </span>
-                    <v-text-field
-                      placeholder="Required"
-                      :model-value="getYesNoValue(ecewe?.confirmation)"
-                      class="summary-value"
-                      density="compact"
-                      flat
-                      variant="solo"
-                      hide-details
-                      readonly
-                      :rules="rules.required"
-                    />
-                  </v-col>
-                </v-row>
-              </v-col>
-              <v-col
-                v-else-if="ecewe?.optInECEWE == 1 && ecewe?.belongsToUnion == 1 && ecewe?.applicableSector == 100000001"
-                cols="12"
-              >
-                <v-row no-gutters>
-                  <v-col cols="12">
-                    <span class="summary-label pt-3">
-                      I confirm our organization/facilities has reached an agreement with the union to amend the
-                      collective agreement(s) in order to implement the ECE Wage Enhancement.
-                    </span>
-                    <v-text-field
-                      placeholder="Required"
-                      :model-value="getYesNoValue(ecewe?.confirmation)"
-                      class="summary-value"
-                      density="compact"
-                      flat
-                      variant="solo"
-                      hide-details
-                      readonly
-                      :rules="rules.required"
-                    />
-                  </v-col>
-                </v-row>
-              </v-col>
-            </v-row>
+            <div v-if="organizationProviderType === ORGANIZATION_PROVIDER_TYPES.GROUP">
+              <v-row v-if="ecewe?.optInECEWE == 1" no-gutters>
```

Comment:
I suspect all these comparisons should use strict equality (===).

Reply:
done